src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1736,6 +1736,10 @@
     <value>Automatically focus pane on mouse hover</value>
     <comment>Header for a control to toggle the "focus follow mouse" setting. When enabled, hovering over a pane puts it in focus.</comment>
   </data>
+  <data name="Globals_DisableMouseZoom.Header" xml:space="preserve">
+    <value>Disable zooming with Ctrl+mouse wheel</value>

```

Comment:
```suggestion
    <value>Change font size with ctrl + mouse wheel</value>
```
The new text avoids the double negative. Personally, I'd prefer to more explicitly mention "font size" than "zoom" because I don't necessarily trust that "zoom" translates well (though I'm not sure how well founded that concern may be). Similarly, I uncapitalized and added spacing around "ctrl + mouse wheel".